app/elements/socobo-inventory/socobo-inventory-create.html

```diff
@@ -204,8 +204,8 @@
             this.item = item;
             this.$.bestBeforeInput.value = item.bestBefore;
             this.$.categoryDropdownMenu.select(this.categories.indexOf(item.category));
-            this.measurements.forEach(function(measurement, index) {
-              if (measurement.value.indexOf(item.measurement.value) !== -1) {
+            this.measurements.forEach(function(measurement, index, arr) {
+              if (arr.indexOf(item.measurement) !== -1) {
```

Comment:
according to MDN the third parameter is the array we are iterating over, so if you are using `Array.prototype.indexOf` there is no need for iterating over this, is there?